lib/fluent/plugin/out_memcached.rb

```diff
@@ -33,12 +31,21 @@ def start
 
   def shutdown
     @memcached.close
+    super
   end
 
   def format(tag, time, record)
     [tag, time, record].to_msgpack
```

Comment:
> what kind of cases should the chunk include tag?

I guess that a one case has as the below config:

```aconf
<match **>
   @type memcached
   <buffer time>
      @type memory
      timekey 1h
      # <snip>
   </buffer>
   # <snip>
</match>
```

And tag is needed in `#write` case.